web-app/js/portal/filter/ui/DateFilterPanel.js

```diff
@@ -101,7 +101,14 @@ Portal.filter.ui.DateFilterPanel = Ext.extend(Portal.filter.ui.BaseFilterPanel,
 
         this.filter.setValue({
             fromDate: this.fromDate.getValue(),
-            toDate: this.toDate.getValue()
+            toDate: this._endOfDay(this.toDate.getValue())
         });
+    },
+
+    _endOfDay: function(date) {
```

Comment:
Can use http://momentjs.com/docs/#/manipulating/end-of/ ?

Except this is a javascript date, not a moment. But maybe we can still use it?